frontend/pages/reviewer.tsx

```diff
@@ -0,0 +1,35 @@
+import { useState } from 'react'
+
+import type { NextPage } from 'next'
+
+import Head from 'next/head'
+import Image from 'next/image'
+import Link from 'next/link'
+import styles from '../styles/Home.module.css'
+
+import BodyCard from '../components/utils/BodyCard'
+
+import { Card, Button } from '@mui/material';
+
+import OngoingReviewsList from '../components/Reviewer/OngoingReviewsList.tsx'
+
+
+const Reviewer: NextPage = () => {
+
+  const [showArchived, setShowArchived] = useState(false);
+  const archivedButtonText = showArchived ? "Hide Archived" : "Show Archvied";
```

Comment:
i have not checked the review page as much (frontend)... I'll continue it in the #14 issue... However, for now, I don't think we'll implement archiving simply because we don't have it as part of the #36 .

This is not really a problem though because archiving (as a feature) only is needed for when there is too much course evaluation assigned to a particular user. Realistically speaking, this will only grow by atmost 4 course evaluation per year for a user. (need to check with @mhodki), but as part of MVP, it is not required